spec/lib/publishing_api_payload/history_spec.rb

```diff
@@ -0,0 +1,110 @@
+RSpec.describe PublishingApiPayload::History do
+  describe "#public_updated_at" do
+    it "returns current time if a major change is published" do
+      freeze_time do
+        first_edition = build(:edition)
+        second_edition = build(:edition, document: first_edition.document, update_type: "major")
+
+        expect(described_class.new(second_edition).public_updated_at).to eq(Time.current)
+      end
+    end
+
+    it "returns most recent change history time if a minor change is published" do
+      first_edition = build(:edition, first_published_at: "2020-02-22 11:00:00")
+      second_edition = build(:edition, document: first_edition.document, update_type: "minor")
+
+      expect(described_class.new(second_edition).public_updated_at).to eq("2020-02-22 11:00:00")
+    end
+  end
+
+  describe "#first_published_at" do
+    it "returns backdated date if edition is backdated" do
+      edition = build(:edition, first_published_at: "2020-02-22 11:00:00", backdated_to: "2020-02-20 09:00:00")
+
+      expect(described_class.new(edition).first_published_at).to eq("2020-02-20 09:00:00")
+    end
+
+    it "returns first published date if edition is not backdated" do
+      edition = build(:edition, first_published_at: "2020-02-22 11:00:00")
+
+      expect(described_class.new(edition).first_published_at).to eq("2020-02-22 11:00:00")
+    end
+  end
+
+  describe "#change_history" do
+    let(:edition) { build(:edition, first_published_at: "2020-02-22 11:00:00") }
```

Comment:
Might be clearer to name this as first_edition

Might also be useful to have a let of `initial_change_note` since you reference the same one a few times in tests.

Reply:
Good points.  I've changed the name of `edition` to `first_edition` and added `initial_change_note`.